lib/rubygems/specification_policy.rb

```diff
@@ -133,16 +134,16 @@ def validate_metadata
       end
 
       if !value.kind_of?(String)
-        error "metadata values must be a String"
+        error "#{entry} value must be a String"
       end
 
       if value.size > 1024
-        error "metadata value too large (#{value.size} > 1024)"
+        error "#{entry} value too large (#{value.size} > 1024)"
```

Comment:
What about to fix this as well? `...value is too large...`

Reply:
Ah, now I see, to improve the grammar of the wording, add "is". 👍